igvm/commands.py

```diff
@@ -794,31 +794,34 @@ def vm_rename(vm_hostname, new_hostname, offline=False):
     """
 
     with _get_vm(vm_hostname) as vm:
-        if vm.dataset_obj['datacenter_type'] != 'kvm.dct':
+        if vm.dataset_obj['datacenter_type'] in ['aws.dct', 'kvm.dct']:
```

Comment:
less is better. identations are bad. please return or raise as early as possible and skip all `else` statements if not really necessary. for example the first if should be negated to raise the error immediately: `if vm.dataset_obj['datacenter_type'] not in ['aws.dct', 'kvm.dct']:`. then leave the `else` away because the execution flow is already stopped.
it makes code so much more readable and easier to understand. whenever you introduce an unnecessary indentation you also put the burden on the "reader" of your code to build this structure in their heads in order to understand it. debugging also becomes much harder.

Reply:
error is now raised in the beginning